tests/e2e/solidity/suites.go

```diff
@@ -1,100 +0,0 @@
-// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-// Implements solidity tests.
-package solidity
-
-import (
-	"fmt"
-	"os/exec"
-
-	"github.com/ava-labs/subnet-evm/plugin/evm"
-	"github.com/ava-labs/subnet-evm/tests/e2e/runner"
-	"github.com/ava-labs/subnet-evm/tests/e2e/utils"
-
-	ginkgo "github.com/onsi/ginkgo/v2"
-	"github.com/onsi/gomega"
-)
-
-const vmName = "subnetevm"
-
-func runHardhatTests(test string) {
-	cmd := exec.Command("npx", "hardhat", "test", test, "--network", "e2e")
-	cmd.Dir = "./contract-examples"
-	out, err := cmd.Output()
-	fmt.Println(string(out))
-	gomega.Expect(err).Should(gomega.BeNil())
-}
-
-// startSubnet starts a test network and launches a subnetEVM instance with the genesis file at [genesisPath]
-func startSubnet(genesisPath string) error {
-	_, err := runner.StartNetwork(evm.ID, vmName, genesisPath, utils.GetPluginDir())
-	gomega.Expect(err).Should(gomega.BeNil())
-	return utils.UpdateHardhatConfig()
-}
-
-// stopSubnet stops the test network.
-func stopSubnet() {
-	err := runner.StopNetwork()
-	gomega.Expect(err).Should(gomega.BeNil())
-}
-
-var _ = utils.DescribePrecompile(func() {
-	ginkgo.It("tx allow list", func() {
-		err := startSubnet("./tests/e2e/genesis/tx_allow_list.json")
```

Comment:
Do we want to keep these tests? I would prefer removing it and adding them back using network-runner and new precompile upgrade tests... There is no need to creating separate networks for each test case, which is slower and inefficient. We can just create one and test with precompile upgrade.

Reply:
To be deprecated in the following PRs

Reply:
Will look into this in the following PR. great idea